backend/server/route/simulation.js

```diff
@@ -16,7 +16,10 @@ router.post("/create", isAuthenticated, (req, res) => {
     return res.json(authorization);
   }
 
-  const { simulation_name } = req.body;
+  const { simulation_title, simulation_desc, simulation_introduction } = req.body;
```

Comment:
I think this can make confusion (Simulation vs Scenario). I know why this happens (due to the integration with both teams. <-> symbol means integration.)
- Backend <-> Frontend: Simulation
- Backend <-> DB: Scenario

I think we need to accept this on this commit, but we need to solve the problem by the talk with Frontend Team since it is hard to change DB structure and its name.